app/DoctrineMigrations/Version20150611105956.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20150611105956 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE unverified_second_factor DROP FOREIGN KEY FK_D79226A2FF3ED4A8');
+        $this->addSql('ALTER TABLE verified_second_factor DROP FOREIGN KEY FK_7964F91CFF3ED4A8');
+        $this->addSql('ALTER TABLE vetted_second_factor DROP FOREIGN KEY FK_29F96B72FF3ED4A8');
+        $this->addSql('ALTER TABLE unverified_second_factor ADD CONSTRAINT fk_unverified_second_factor_identity FOREIGN KEY (identity_id) REFERENCES identity (id) ON DELETE CASCADE');
```

Comment:
Please verify if just dropping the (as far as I can see) unneeded FK's is enough